docs/clipboad.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Get/Set Clipboard
+---
+
+## Get Clipboard
+
+For iOS 13+ real devices, Apple security preferences require the WebDriverAgentRunner application to be in foreground in order to be able to receive the system clipboard content.
```

Comment:
this is also true for simulators, but there we could use simctl to workaround this limitation